app/src/components/nftCard.tsx

```diff
@@ -97,9 +97,9 @@ export const NftCard = (props: NftCardProps): React.ReactElement => {
                   </Box>
                 )}
                 <Stack.Item growthFactor={1} shrinkFactor={1}>
-                  {collectionUrl ? (
+                  {collectionUrl ? collection.name !== 'null' && (
```

Comment:
you need to fix this in the client. the name should be Optional, it should never be the string "null"

Reply:
cool, now that its fixed in the client pls update this too.